server/jetstream_api.go

```diff
@@ -3987,6 +4015,10 @@ func (s *Server) jsConsumerCreateRequest(sub *subscription, c *client, a *Accoun
 	}
 	resp.ConsumerInfo = o.initialInfo()
 	s.sendAPIResponse(ci, acc, subject, reply, string(msg), s.jsonResponse(resp))
+
+	if time.Now().Before(o.cfg.PauseUntil) {
```

Comment:
We can certainly add `IsZero` checks, but strictly speaking, any time "in the past" also counts as unpaused so it won't gain us much if the consumer has ever been paused and not manually cleared/reset back to a zero timestamp before.

We also don't clear the pause state from the consumer config once the pause elapses because that would mean the server has to generate config updates by itself, which I don't think we do anywhere else and might race against something the user is trying to do.

Reply:
Done!

Reply:
We do, it is rare, but we do have system propose updates. consumer InactiveThreshold and stream moves.